src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import {
+	Box,
+	Typography,
+	ThemeProvider,
+} from '@mui/material'
+import { Link } from 'react-router-dom'
+import { useForm } from 'react-hook-form'
+import { DevTool } from '@hookform/devtools'
+
+import {Button as LoginButton} from '../../../components/PublicFormButton'
+import palette from '../../../theme/palette'
+
+import { theme } from './styles'
+import { EmailTextField } from './EmailTextField'
+import { PasswordTextField } from './PasswordTextField'
+import { RememberMeCheckbox } from './RememberMeCheckbox'
+import { LoginInput } from './LoginInput'
+//HACK Where should I put LoginInput Type to ?
```

Comment:
??

Reply:
it's a call for attention, so it served its purpose